wordpress-mobile/gutenberg-mobile
# Update video player to open the URL by browser on Android

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1119

gutenberg PR: https://github.com/WordPress/gutenberg/pull/16089

This should change only Android. We are canceling out the react-native-video's player controls and replacing it with system browser:

![video-android](https://user-images.githubusercontent.com/5032900/59274593-52c7fd80-8c63-11e9-9bbd-165b69af49d5.gif)

iOS should remain same:

![video-ios](https://user-images.githubusercontent.com/5032900/59274611-5ce9fc00-8c63-11e9-963a-360aba2c27ed.gif)

To test:

WPAndroid:

- Open a post with video
- Verify that videos have a single play button and controls are invisible
- Tap on the video
- Verify that video opens outside of the app, probably in browser(I think it might also be opened by another app if user has selected it as the default app to open videos)

WPiOS should remain same:

- Open a post with video
- Tap on the video
- Verify that video opens fullscreen INSIDE the app

Update release notes:
